### ISSUE TITLE
Build command not working

### ISSUE THREAD

**Issue by psociety** [mathishouis/scuti-renderer#37]
```
D:\projects\scuti-renderer-dev>npm run build

> scuti-renderer@1.0.0 build
> parcel build public/index.html

× Build failed.

@parcel/namer-default: Target "main" declares an output file path of "dist\index.js" which does not match the compiled bundle type "html".

  D:\projects\scuti-renderer-dev\package.json:5:11
    4 |   "description": "",
  > 5 |   "main": "dist/index.js",
  >   |           ^^^^^^^^^^^^^^^ Did you mean "dist\\index.html"?
    6 |   "source": "src/index.ts",
    7 |   "scripts": {

  ℹ Try changing the file extension of "main" in package.json.
```

If i change it :

```
D:\projects\scuti-renderer-dev>npm run build

> scuti-renderer@1.0.0 build
> parcel build public/index.html

× Build failed.

@parcel/core: Unexpected output file type .html in target "main"

  D:\projects\scuti-renderer-dev\package.json:5:11
    4 |   "description": "",
  > 5 |   "main": "dist/index.html",
  >   |           ^^^^^^^^^^^^^^^^^ File extension must be .js, .mjs, or .cjs
    6 |   "source": "src/index.ts",
    7 |   "scripts": {

  ℹ The "main" field is meant for libraries. If you meant to output a .html file, either remove the "main" field or choose a different target name.
  ℹ Learn more: https://parceljs.org/features/targets/#library-targets

```

Parcel disagrees with itself :S

The only way it works is if i simply remove "main" attr from package.json

**Comment by psociety** [comment]
The package can't be imported as package on other projects because it's build as a deployable dist rather than as package. It must be refactored or otherwise it will never load on a client package : o

**Comment by Alynva** [comment]
I think that you should change the build script, and not the main property. But the module build command is `npm run tsc-build`

**Comment by psociety** [comment]
@Alynva but the output right now doesn't make any sense, it's a renderer, it's not expected to run standalone :S

**Comment by mathishouis** [comment]
It could be nice to replace the build script by the tsc-build script, as the current build script is not really useful. It will avoid some confusions 😸

**Comment by tetreum** [comment]
I've already did this but the PR is not ready yet as changing the build script was just a part of adding storybook. This way stuff like  `script1.js`, `script2.js`, `script3.js` will have a proper debugging/preview space.

**Comment by mathishouis** [comment]
Fixed in #39
